src/bindings/python/src/openvino/runtime/opset10/__init__.py

```diff
@@ -11,7 +11,7 @@
 from openvino.runtime.opset1.ops import add
 from openvino.runtime.opset1.ops import asin
 from openvino.runtime.opset4.ops import asinh
-from openvino.runtime.opset3.ops import assign
+from openvino.runtime.opset6.ops import assign
```

Comment:
@tomdol @mitruska is it OK to change already established opset versions?

Reply:
In fact this change aligns the Python opsets with C++ where the opset10 contains Assign v6, not v3 https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/opsets/opset10_tbl.hpp#L161